mtags-shared/src/main/scala/scala/meta/internal/pc/AutoImportPosition.scala

```diff
@@ -26,11 +28,18 @@ object AutoImportPosition {
   // class Main {
   //   def foo<COMPLETE> // inferred indent is 2 spaces.
   // }
+  @tailrec
   def inferIndent(lineStart: Int, text: String): Int = {
     var i = 0
     while (lineStart + i < text.length && text.charAt(lineStart + i) == ' ') {
       i += 1
     }
-    i
+
+    val pos = lineStart + i
+    val accepted = Set('\n', '\r')
```

Comment:
Thanks for making sure that the support is cross-platform! :+1: Small comment, it seems that this collection will be re-allocated on each invocation of the recursive `inferIndent` function: maybe we could extract it to a static constant and give it a more descriptive name (like `endOfLineCharacters` or some such). :slightly_smiling_face: